mulighetsrommet-api/src/test/kotlin/no/nav/mulighetsrommet/api/clients/cms/CmsClientTest.kt

```diff
@@ -0,0 +1,116 @@
+package no.nav.mulighetsrommet.api.clients.cms
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.collections.shouldHaveSize
+import io.mockk.clearMocks
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.mockk
+import kotlinx.serialization.json.Json
+import no.nav.mulighetsrommet.api.clients.sanity.SanityClient
+import no.nav.mulighetsrommet.api.domain.dto.SanityResponse
+import no.nav.mulighetsrommet.api.services.cms.CacheUsage
+import no.nav.mulighetsrommet.api.services.cms.SanityService
+import java.util.*
+
+class CmsClientTest : FunSpec({
```

Comment:
Skulle denne hete `SanityServiceTest` for å matche klassen den i hovedsak tester?